connect.yaml

```diff
@@ -6,22 +6,31 @@ deployAs:
       standardConfiguration:
         - key: CTP_PROJECT_KEY
           description: Commercetools project key
+          required: true
         - key: CTP_CLIENT_ID
           description: Commercetools client ID
+          required: true
         - key: CTP_SCOPE
           description: Commercetools client scope
+          required: true
         - key: CTP_AUTH_URL
           description: Commercetools Auth URL
+          required: true
         - key: CTP_API_URL
```

Comment:
we can add europe gcp region as default value.

Reply:
https://github.com/commercetools/connect-payment-integration-template/pull/171/commits/9070c015e2af5f71a21f37640635bcb18e0aef8d